app/models/alchemy/page.rb

```diff
@@ -121,6 +121,8 @@ class Page < BaseRecord
     has_one :draft_version, -> { drafts }, class_name: "Alchemy::PageVersion"
     has_one :public_version, -> { published }, class_name: "Alchemy::PageVersion", autosave: -> { persisted? }
 
+    has_many :page_ingredients, class_name: "Alchemy::Ingredients::Page", foreign_key: :related_object_id, dependent: :nullify
```

Comment:
TIL There is also a `ON DELETE SET NULL` for foreign keys. But this would make it necessary to create a migration.

Reply:
I don't think we can use the DB in this case, because `related_object_id` is polymorphic and could relate to anything :)

Reply:
Ah, yes. We do not even have a foreign key because of this reason.